modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItIndexSpoolTest.java

```diff
@@ -57,10 +58,11 @@ protected void cleanUp() {
     /**
      * Test.
      */
+    @Disabled("https://issues.apache.org/jira/browse/IGNITE-17959")
     @ParameterizedTest(name = "tableSize=" + ARGUMENTS_PLACEHOLDER)
-    @ValueSource(ints = {1, 10, IN_BUFFER_SIZE, IN_BUFFER_SIZE + 1, 2000})
+    @ValueSource(ints = {1, 10, IN_BUFFER_SIZE * 1/**partitions count*/, IN_BUFFER_SIZE * 1 + 1, 2000})
```

Comment:
maybe introduce new constant in the class for the purpose?

Reply:
Thanks for pointing, i change it, now there are 2 arguments passed int the test, rowCount and partitions